content/cucumber/hooks.md

```diff
@@ -3,23 +3,24 @@ menu:
 - reference
 source: https://github.com/cucumber/cucumber/wiki/Hooks/
 title: Hooks
+polyglot: true
 ---
 
-> TODO: Important. Generalize.
+Cucumber provides a number of Hooks which allow us to run blocks of code at various points in the Cucumber test cycle.
 
-Cucumber provides a number of Hooks which allow us to run blocks at various points in the Cucumber test cycle. 
-You can put them in your `support/env.rb` file, or any other file under the `support` directory 
-(for example, in a file called `support/hooks.rb`). 
+{{% text "ruby" %}}
+You can put them in your `support/env.rb` file, or any other file under the `support` directory
+(for example, in a file called `support/hooks.rb`).
+{{% /text %}}
 
-There is no association between where a Hook is defined and which Scenario or Step it is run for. 
-But if you want more fine grained control, you can use Tagged Hooks (see below).
+There is no link between where a Hook is defined and which Scenario or Step it is run for.
+If you want more fine grained control, you can use [Tagged Hooks](#tagged-hooks).
 
-All defined Hooks are run whenever the relevant event occurs.
+All defined Hooks run whenever the relevant event occurs.
 
 ## Scenario hooks
 
-`Before` hooks will be run before the first Step of each Scenario. 
-They are run in the same order they are registered.
+`Before` hooks will be run before the first Step of each Scenario. They will run in the **same order** in which they are registered.
```

Comment:
It seems that you want to avoid the indirectness of the expression `will be run`.

The first sentence of this paragraph still uses that turn of phrase.